ElrondNetwork/elrond-go
# fix account query options for altered accounts

## Reasoning behind the pull request
- account query options were not set correctly when fetching the altered accounts
  
## Proposed changes
- set all available options, since the data is there

## Testing procedure
full system test + test altered accounts as follows:
- find a block that has altered accounts (let's say nonce 50)
- query `<node-ip>:<node-port>/block/altered-accounts/50`
- choose a random address and query `<node-ip>:<node-port>/address/<address from the previous step>?blockNonce=50` - the balance should be identical to the one obtained in the previous step
- search a block where the same address had another transaction and query the same 2 endpoints, looking for the exact same balance

## Pre-requisites

Based on the [Contributing Guidelines](https://github.com/ElrondNetwork/elrond-go/blob/master/.github/CONTRIBUTING.md#branches-management) the PR author and the reviewers must check the following requirements are met:
- was the PR targeted to the correct branch?
- if this is a larger feature that probably needs more than one PR, is there a `feat` branch created?
- if this is a `feat` branch merging, do all satellite projects have a proper tag inside `go.mod`?
